lms/djangoapps/edraak_sendinblue/api_client.py

```diff
@@ -33,5 +33,30 @@ def create_contact(username, email, name, blacklisted):
             log.exception("Exception when calling SendInBlue for email ({}) ContactsApi->create_contact: {}\n".format(email, e))
 
 
-# TODO: update contact attributes upon completion of profile
-# TODO: delete a contact and integrate with user retirement
\ No newline at end of file
+def update_contact(email):
+    configuration = setup_sendinblue_configuration()
+
+    if configuration:
+        api_instance = sib_api_v3_sdk.ContactsApi(sib_api_v3_sdk.ApiClient(configuration))
+        email = email
+        updated_contact = sib_api_v3_sdk.UpdateContact(email=email)
```

Comment:
Ok here you need to add subscription status through `email_blacklisted`, name using `attributes`. Check this example: https://developers.sendinblue.com/docs/synchronise-contact-lists#update-your-contact

In any case, I just noticed on `create_contact` the attr `update_enabled=True` which means if the contact exists, update their fields. So its safe to use `create_contact` inside `change_sendinblue_user_state`. The functions you built `update_contact` and `delete_contact` are good. Just fix `update_contact` and change the other file.